docs/user_guide/branding.rst

```diff
@@ -77,41 +77,43 @@ To reference an external website, make sure your link starts with ``http``:
        }
    }
 
-Customize logo alternative text
--------------------------------
-
-You may set a custom ``alt text`` for your logo to replace the default ``"logo image"`` generic description.
-Adding a descriptive ``alt text`` can help make your documentation more accessible to readers using screen readers or another assistive tech.
+Add a logo title
+----------------
 
-To do so, customize the ``html_theme_options["logo"]["alt_text"]`` configuration option as in the following example:
+To add a title in the brand section of your documentation, define a value for ``html_theme_options.logo["text"]``.
+This title will appear next to the logo image if set.
 
 .. code-block:: python
-   :caption: conf.py
 
    html_theme_options = {
        "logo": {
-           # Because the logo is also a homepage link, including "home" in the alt text is good practice
-           "alt_text": "My Project Name - Home",
+           "text": "My awesome documentation",
        }
    }
 
-Add a logo title
-----------------
+.. note:: The ``html_title`` field will work as well if no logo images are specified.
 
-To add a title in the brand section of your documentation, define a value for ``html_theme_options.logo["text"]``.
-This title will appear next to the logo image if set.
+Customize logo alternative text
+-------------------------------
+
+Adding a descriptive ``alt text`` can help make your documentation more accessible to readers using screen readers or other assistive tech.
+
+By default, the theme will set ``alt text`` to ``<Your Project Name> - Home`` unless you provide a logo title, in which case the theme sets the ``alt text`` to the empty string. The assumption is that if you provide a logo title, the title is probably doing the work of the alt text.
+
+However, if you don't like the default values, you may set a custom ``alt text`` for your logo.
+
+To do so, customize the ``html_theme_options["logo"]["alt_text"]`` configuration option as in the following example:
 
 .. code-block:: python
+   :caption: conf.py
 
    html_theme_options = {
        "logo": {
-           "text": "My awesome documentation",
+           # Because the logo is also a homepage link, including "home" in the alt text is good practice
+           "alt_text": "My Project Name - Home",
```

Comment:
Good point. Hmm. This would result in assistive tech representing the link as "PggPy - Home A pretty good geometry package."

I think that this section of the docs needs to be rewritten to provide a handful of examples (maybe a couple ones marked good and a couple marked bad).

That was something I was planning to do in a follow-up PR.

Reply:
does it help if default is "Home - pkgname" instead of "pkgname - Home"?

Reply:
After examining logo alt text on a few sites, such as [pac.bz](https://www.pac.bz/), [tink.uk](https://tink.uk/), [sarasoueidan.com](https://www.sarasoueidan.com/), and [w3.org](https://w3.org), I do not think there's a clear consensus on how to do this. 

So I think, ultimately, what we have is fine.